arch/xtensa/include/xtensa-asm2-context.h

```diff
@@ -88,7 +89,8 @@
 	(BASE_SAVE_AREA_SIZE_COMMON + \
 	 BASE_SAVE_AREA_SIZE_LOOPS + \
 	 BASE_SAVE_AREA_SIZE_SCOMPARE + \
-	 BASE_SAVE_AREA_SIZE_THREADPTR)
+	 BASE_SAVE_AREA_SIZE_THREADPTR + \
```

Comment:
It seems like you are inserting `EXCCAUSE` before `SCOMPARE`. Please keep this list ordered so it is easier to tell the ordering of these fields.